devfile.yaml

```diff
@@ -14,7 +14,7 @@ components:
     id: che-machine-exec-plugin:0.0.1
   - name: che-dev
     type: kubernetes
-    local: .che-dev.yaml
+    rerefence: .che-dev.yaml
```

Comment:
It is not related to this PR, just a question: Should it be still hidden even if devfile is not? I mean should not this file be renamed to `che-dev.yaml`

Reply:
I dunno. Author is next seat and we can ask him :)